.pkgmeta

```diff
@@ -19,6 +19,7 @@ externals:
   WeakAuras/Libs/LibCustomGlow-1.0: https://github.com/Stanzilla/LibCustomGlow
   WeakAuras/Libs/LibDBIcon-1.0: https://repos.wowace.com/wow/libdbicon-1-0/trunk/LibDBIcon-1.0
   WeakAuras/Libs/LibGetFrame-1.0: https://github.com/mrbuds/LibGetFrame
+  WeakAuras/Libs/Archivist: https://github.com/emptyrivers/Archivist
```

Comment:
> store = Archivist:CloneStore(store, openStore)

Yeah, that's an oversight in the documentation (also the signature in the docs is wrong, it should be `function Archivist:CloneStore(store, newId, openStore)`. The <Verb>Store methods just forward directly to <Verb>, though. So it does return the storeID.

> Archivist:DeleteStore(store, force)

The force parameter is necessary if, for example, a store type which is not used anymore needs cleanup. Since the store type isn't registered, most of the verbs will refuse to operate on the data. But, you may still want to *delete* data, so I included an escape hatch.

That's a bit of unnecessary complexity, but my thinking at the time was to allow prototypes to include a Delete method, which might be useful to delete sub-stores at the same time. Thus, it would make sense for Delete to refuse to delete stores if the storetype isn't registered. But I never implemented that, so the force flag currently is redundant.

> Archivist:DeleteAll(storeType)

I debated on including this API at all, since it's pretty dangerous in any case. But you're right, that is an oversight.

Reply:
The point about the DeleteStore method is, that this the variant that takes a store. To get a store, you would have to succesfully called :Open (if I see that correctly). To succesfully call :Open, the storageType must be registered. But if it's registered, the force parameter can't be needed.

Reply:
Yes, I understand that now, thanks. I'll drop the force flag.